README.md

```diff
@@ -1,45 +1,50 @@
 ![Banner](./GitHub_Banner.png)
 
-## TL;DR
-Finamp is a free Jellyfin music player for Android, iOS and Desktop. It allows you to Stream and Download the music you own. The design is mimicking popular music streaming services but still with a unique twist and a few customizations and of course privacy!
-> [!IMPORTANT]
-> You do **need** access to a [jellyfin](https://jellyfin.org) server else you **wont be able to use Finamp at all**.
-
-## Download now
 <div align="center">
+
 [<img src="assets/app-store-badges/app-store.svg"
     alt="Download on the App Store"
-    height="60">](https://apps.apple.com/us/app/finamp/id1574922594)
+    height="60">](https://apps.apple.com/us/app/finamp/id1574922594) 
 [<img src="assets/app-store-badges/fdroid.png"
     alt="Get it on F-Droid"
-    height="80">](https://f-droid.org/packages/com.unicornsonlsd.finamp/)
+    height="80">](https://f-droid.org/packages/com.unicornsonlsd.finamp/) 
 [<img src="assets/app-store-badges/play-store.png"
     alt="Get it on Google Play"
     height="80">](https://play.google.com/store/apps/details?id=com.unicornsonlsd.finamp)
 </div>
 
+## TL;DR
+Finamp is a **free** Jellyfin music player for Android, iOS and Desktop. It allows you to Stream and Download the music you own. It has a **modern design** which may remind you of other popular music streaming apps but with a unique twist and a few **customizations** and of course **privacy**!
```

Comment:
the `which may remind you of other popular music streaming apps but with a unique twist` part is a bit too long for my taste. How about `it has a **modern design** that (is unique but) should feel familiar right-away`? The parts in parentheses `( )` are optional, you can decide if you want to keep that, just don't keep the parentheses themselves ^^